### DISCUSSION TITLE
Can't download videos

### DISCUSSION THREAD

**Discussion by tropicalremix** [aclap-dev/video-downloadhelper#875] [Help and Q&A]
### Your question

I was wondering how your supposed to download movies with the new update. I hit download when it shows up in the app but nothing happens
I am using firefox. When the add-on shows the video with the download button on it. I click on the download. It just buffers for a few seconds than goes back to the download button again.

### Addon & CoApp version and details

I am using this add-on in firefox. Version 9.0.2.3. I have downloaded the CoApp but it keeps saying outdated

**Comment by GrampaWildWilly** [comment]
This is a duplicate of #872.  You must not do this.  I will not help you here.  Go back to the other discussion & let's deal with things there.
